### ISSUE TITLE
Replace `JSBI` with native `BigInt`

### ISSUE THREAD

**Issue by arthurschreiber** [tediousjs/tedious#1468]
We used JSBI as a polyfill for `BigInt` on older Node.js versions. All versions of Node.js that we currently support also support `BigInt` out-of-the box, so we can drop the dependency on that.

**Comment by MichaelSun90** [comment]
Hi @arthurschreiber , I am current working on this, and had already replace the places that uses JSBI with BigInt. Found something with in  writable-tracking-buffer.ts:
```js
 writeBigInt64LE(value: number) {
    this.writeBigU_Int64LE(value);
  }

  writeInt64LE(value: number) {
    this.writeBigInt64LE(value);
  }

  writeUInt64LE(value: number) {
    this.writeBigUInt64LE(value);
  }

  writeBigUInt64LE(value: number) {
    this.writeBigU_Int64LE(value);
  }

  private writeBigU_Int64LE(value: number) {
```
I move the functions a bit, so this group function can be compared together. For writeInt64LE, it calls writeBigInt64LE which calls writeBigU_Int64LE. Is there a specific reason that we cannot directly call writeBigU_Int64LE which holds the actual logic. Also, I found that  writeBigUInt64LE is not called anywhere else. Not sure why we need to keep this function. If there is no specific reason for this, I can refactor this part as well.
